pkg/apis/kubevirt/v1/types.go

```diff
@@ -69,13 +69,15 @@ type VersionSpec struct {
 }
 
 type ComponentSpec struct {
-	Version string `json:"version,omitempty"`
-	UseKVM  bool   `json:"useKVM"`
+	Version                 string `json:"version,omitempty"`
+	UseKVM                  bool   `json:"useKVM"`
+	NumberOfLabellerRetries int    `json:"numberOfLabellerRetries,omitempty"`
 }
 
 type TemplateValidatorSpec struct {
 	Version                   string `json:"version,omitempty"`
 	TemplateValidatorReplicas int    `json:"templateValidatorReplicas,omitempty"`
+	NumberOfValidatorRetries  int    `json:"numberOfLabellerRetries,omitempty"`
```

Comment:
Typo: should be `numberOfValidatorRetries`.

I'd actually use `MaxRetries` or `MaxRetriesNumber` for both structs.